front/app/containers/Admin/projects/project/permissions/components/UserSearch.tsx

```diff
@@ -59,13 +60,29 @@ function isModerator(user: IGroupMembershipsFoundUserData) {
   return get(user.attributes, 'is_moderator') !== undefined;
 }
 
+interface UserOption extends IOption {
+  email: string;
+  isAdmin: boolean;
+}
+
 const UserSearch = memo(
   ({ projectId, intl: { formatMessage } }: Props & WrappedComponentProps) => {
-    const [selection, setSelection] = useState<IOption[]>([]);
+    const [selection, setSelection] = useState<UserOption[]>([]);
     const [loading, setLoading] = useState(false);
     const [processing, setProcessing] = useState(false);
     const [searchInput, setSearchInput] = useState('');
     const moderators = useProjectModerators(projectId);
+    const [showModal, setShowModal] = useState(false);
+    const closeModal = () => {
+      setShowModal(false);
+    };
+    const openModal = () => {
+      setShowModal(true);
+    };
+    // If it is an admin, that isn't counted as a new seat to buy
+    const noOfSeatsToBuy = selection.filter(
+      (selectedUser) => !selectedUser.isAdmin
+    ).length;
```

Comment:
This seems not correct. If I have 2 available collaborator seats and want to add 1 user as a project moderator, noOfSeatsToBuy is 1. But I still have 2 open spots, so this should be 0, right?

Reply:
Edit: the value seems to be not really used (only as a condition that's always true and as a value in a string when we want to buy extra seats), so the value sometimes being wrong didn't cause bugs for now.

Reply:
The logic is correct but the naming might be confusing. Seats to buy means seats that count to your license. This is closely linked to the concept of admins so I'll make the change there to a clearer or remove depending on solution